packages/synthetix-governance/contracts/interfaces/ISynthetixElectionModule.sol

```diff
@@ -0,0 +1,45 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IElectionModule as IBaseElectionModule} from "@synthetixio/core-modules/contracts/interfaces/IElectionModule.sol";
+
+interface ISynthetixElectionModule is IBaseElectionModule {
```

Comment:
This is the SNX specific ElectionModule interface, that extends the generic one.